rust-src/concordium_base/src/id/types.rs

```diff
@@ -2779,6 +2804,11 @@ pub struct CredentialValidity {
 /// Attribute value as represented in the identity attribute credential values. An attribute value has ither been committed to,
 /// revealed, or just proven known.
 #[derive(Debug, PartialEq, Eq, SerdeSerialize, SerdeDeserialize, Clone)]
+#[serde(bound(
+    serialize = "C: Curve, AttributeType: Attribute<C::Scalar> + SerdeSerialize",
+    deserialize = "C: Curve, AttributeType: Attribute<C::Scalar> + SerdeDeserialize<'de>"
+))]
+#[serde(rename_all = "camelCase", tag = "repr", content = "value")]
```

Comment:
Mostely we use `type` when including the `tag` in the JSON representation of an enum in the code base.

```suggestion
#[serde(rename_all = "camelCase", tag = "type", content = "value")]
```

Reply:
I didn't use the name type, because it is not so much a type of an attribute, it is more how it is represented.